### PR TITLE
Feature/comment

### PR DESCRIPTION
**Pull request by TynchtykbekAkmatov**

### CODE REVIEW

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+@RequiredArgsConstructor
+@RequestMapping("api/comment")
+@CrossOrigin(origins = "*", maxAge = 3600)
+@Tag(name = "Comment API", description = "All endpoints of comment")
+public class CommentApi {
+
+    private final CommentService commentService;
+
+    @Operation(summary = "Save comment", description = "Save new comment")
+    @PostMapping("/save-comment/{cardId}")
```
Comment (DatkaM) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:24]: convention

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    public CommentResponse saveComment(@PathVariable Long cardId,
+                                       @RequestBody CommentRequest request){
+        return commentService.saveComment(cardId, request);
+    }
+
+    @Operation(summary = "Edit comment", description = "Edit comment by id")
+    @PutMapping("/update/{id}")
```
Comment (DatkaM) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:31]: convention

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    public CommentResponse updateComment(@PathVariable Long id,
+                                         @RequestBody CommentRequest request){
+        return commentService.editComment(id, request);
+    }
+
+    @Operation(summary = "Delete comment", description = "Delete comment by id")
+    @DeleteMapping("/delete/{id}")
```
Comment (DatkaM) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:38]: convention

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    public SimpleResponse deleteComment(@PathVariable Long id){
+        return commentService.deleteComment(id);
+    }
+
+    @Operation(summary = "Get all comments", description = "Get all comments by card id")
+    @GetMapping("/get-all/{id}")
```
Comment (DatkaM) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:44]: convention, you can write "comments" or not write anything at all

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    @Operation(summary = "Delete comment", description = "Delete comment by id")
+    @DeleteMapping("{id}")
+    public SimpleResponse deleteComment(@PathVariable Long id){
+        return commentService.deleteComment(id);
+    }
+
+    @Operation(summary = "Get all comments", description = "Get all comments by card id")
+    @GetMapping("{id}")
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:44]: card/{id}

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+@RequiredArgsConstructor
+@RequestMapping("api/comments")
+@CrossOrigin(origins = "*", maxAge = 3600)
+@Tag(name = "Comment API", description = "All endpoints of comment")
+public class CommentApi {
+
+    private final CommentService commentService;
+
+    @Operation(summary = "Save comment", description = "Save new comment")
+    @PostMapping("{cardId}")
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:24]: card/{id}

**Comment by ChyngyzIshen** [comment]
Good job!

**Pull request merged**